xdsl/dialects/irdl.py

```diff
@@ -192,3 +169,19 @@ def verify_(self) -> None:
             raise ValueError("name attribute is required")
         if not isinstance(self.attributes["name"], StringAttr):
             raise ValueError("name attribute must be a string attribute")
+
+
+IRDL = Dialect([
+    DialectOp,
+    ParametersOp,
+    TypeOp,
+    ConstraintVarsOp,
+    OperandsOp,
+    ResultsOp,
+    OperationOp,
+], [
+    EqTypeConstraintAttr, AnyTypeConstraintAttr, AnyOfTypeConstraintAttr,
```

Comment:
why are these not following the same format as lines 174 to 181?

Reply:
This is something yapf does automatically for some reason. This might be another reason to move to a different formatter?

I think I could also solved this by adding some comments, similar to the Arith case.